chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,615 @@
+\index{instrumental variables}
+designs, unlike the previous approaches,
+assume that the treatment effect is not directly identifiable.
+Similar to RD designs,
+IV designs focus on a subset of the variation in treatment take-up.
+Where RD designs use a ``sharp'' or binary cutoff,
+IV designs are ``fuzzy'', meaning that the input does not completely determine
+the treatment status, but instead influence the \textit{probability of treatment}.
+
+You will need variables in your data
+that can be used to estimate the probability of treatment for each unit.
+These variables are called \textit{instruments}.
+Testing that a variable is a valid instrument is a non-trivial and important task
+that is outside the scope of this book.
+In IV designs, instead of the ordinary regression estimator,
+a special version called two-stage least squares (2SLS) is used
+to estimate the impact of the treatment.
+Stata has a built-in command called \texttt{ivregress},
+and another popular implementation is the user-written command \texttt{ivreg2}.
+
+%%%%% Matching
+
+\textbf{Matching}\sidenote{
+	\url{https://dimewiki.worldbank.org/Matching}}
+methods use observable characteristics to construct
+sets or pairs of treatment and control units
+so that the observations in each set are as similar as possible. \index{matching}
+These sets can either consist of exactly one treatment and one control observation (one-to-one),
+or it can be a set observations where either
+both groups have more than one observation represented (many-to-many),
+or where only one group has more than one observation included (one-to-many).
+
+The matching can even be done before the random assignment,
+so that treatment can be randomized within each matching set.
+This is a type of experimental design.
+Furthermore, if no control observations were identified before the treatment,
+then matching can be used to ex-post identify a control group,
+by finding the observations that are the most similar to the treated group.
+It is very difficult to test the validity of an ex-post matching
+as one would have to show that the difference in outcome is
+due to the impact of the treatment and not due to the groups not being similar enough.
+
+A valid matching must be made on data that is not related to the treatment
+or anything that the treatment could have affected in any way.
+Many matching algorithms can only match on a single variable,
+so you first have to turn many variables into a single varaible
+by using an index or a propensity score.\sidenote{
+	\url{https://dimewiki.worldbank.org/Propensity\_Score\_Matching}}
+The \texttt{iematch}\sidenote{
+	\url{https://dimewiki.worldbank.org/iematch}}
+command in the \texttt{ietoolkit} package developed by DIME Analytics
+produces matchings based on a single continuous matching variable.
+
+%-----------------------------------------------------------------------------------------------
+
+
+\subsection{One observation or multiple observations over time}
```

Comment:
Any suggestion on what to call this section? I went back and forth on many examples:
* _Recording observations over time_ - Sounds better than current title but while it cover most of what this section is about, it does not cover everything
* _Longitudinal vs cross section data_ - Correct but using technical terms we have not explained yet

Any other suggestions?

Reply:
what about "Structuring datasets over multiple time periods"?

Reply:
Better than both my suggestions. My only concern is that it does not cover _cross-section data_ that is also a part of this section. But since most of this section is about longitudinal I still went ahead and updated to your suggestion. We can update later if needed, and this is much better than what I had. 51e9ec201901bed2ce0557bb3a691d6820f75735